Code deprecation errors

```
 Error: Usage of deprecated trait Drupal\Tests\field\Traits\EntityReferenceTestTrait in class Drupal\Tests\localgov_workflows\Kernel\WorkflowsInstallTest:
in drupal:10.2.0 and is removed from drupal:11.0.0. Use
   \Drupal\Tests\field\Traits\EntityReferenceFieldCreationTrait instead.
 ------ ----------------------------------------------------------------- 
  Line   tests/src/Kernel/WorkflowsInstallTest.php                        
 ------ ----------------------------------------------------------------- 
  20     Usage of deprecated trait                                        
         Drupal\Tests\field\Traits\EntityReferenceTestTrait in class      
         Drupal\Tests\localgov_workflows\Kernel\WorkflowsInstallTest:     
         in drupal:10.2.0 and is removed from drupal:11.0.0. Use          
            \Drupal\Tests\field\Traits\EntityReferenceFieldCreationTrait  
         instead.                                                         
 ------ ----------------------------------------------------------------- 


Error:  [ERROR] Found 1 error                                                          
```

--- a/tests/src/Functional/RequireLogTest.php
+++ b/tests/src/Functional/RequireLogTest.php
@@ -67,7 +67,7 @@ class RequireLogTest extends BrowserTestBase {
     // Enabling field_ui will change that button.
     $this->submitForm([
       'options[revision_required]' => TRUE,
-    ], 'Save content type');
+    ], 'Save');
 
     $this->drupalGet('/node/add/require_log_test');
     $log_field = $this->assertSession()->fieldExists('revision_log[0][value]');
@@ -79,7 +79,7 @@ class RequireLogTest extends BrowserTestBase {
     $this->assertTrue($checkbox->isChecked());
     $this->submitForm([
       'options[revision_required]' => FALSE,
-    ], 'Save content type');
+    ], 'Save');
 
     $this->drupalGet('/node/add/require_log_test');
     $log_field = $this->assertSession()->fieldExists('revision_log[0][value]');
@@ -91,7 +91,7 @@ class RequireLogTest extends BrowserTestBase {
     $this->assertFalse($checkbox->isChecked());
     $this->submitForm([
       'options[revision_required]' => TRUE,
-    ], 'Save content type');
+    ], 'Save');
 
     $this->drupalGet('/node/add/require_log_test');
     $log_field = $this->assertSession()->fieldExists('revision_log[0][value]');
@@ -103,7 +103,7 @@ class RequireLogTest extends BrowserTestBase {
     $this->assertTrue($checkbox->isChecked());
     $this->submitForm([
       'options[revision]' => FALSE,
-    ], 'Save content type');
+    ], 'Save');
     $this->drupalGet('/node/add/require_log_test');
     $log_field = $this->assertSession()->fieldNotExists('revision_log[0][value]');
     $this->drupalGet('/admin/structure/types/manage/require_log_test');

--- a/tests/src/Kernel/WorkflowsInstallTest.php
+++ b/tests/src/Kernel/WorkflowsInstallTest.php
@@ -17,6 +17,9 @@ use Drupal\workflows\Entity\Workflow;
 class WorkflowsInstallTest extends KernelTestBase {
 
   use ContentTypeCreationTrait;
+  // FIXME: Replace with EntityReferenceFieldCreationTrait when Drupal 10.1 is
+  // end of life.
+  // @phpstan-ignore-next-line.
   use EntityReferenceTestTrait;
   use NodeCreationTrait;
   use PathautoTestHelperTrait;

